### PR TITLE
8209 questionnaire rich text

### PR DESCRIPTION
**Pull request by arisah-ijeoma**
I got some failing spec here because of the summernote editor and wouldn't mind pairing on it

Otherwise, summernote works

We might need to hasten up the page load

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,6 +20,11 @@ gem 'uri-js-rails' # URI manipulation
 gem 'bootstrap-datepicker-rails'
 gem 'paperclip', '~> 5.2'
 
+gem 'turbolinks', '~> 5.0.0'
```
Comment (meltheadorable) [inline Gemfile]: Is turbolinks really required? We removed this for good reasons, unfortunately, and I don't trust that adding it back will be consequences-free.

Comment (arisah-ijeoma) [reply Gemfile]: I'll remove it an see if we can go without

**Comment by arisah-ijeoma** [comment]
I can not find `Edit Response` on the page

**Comment by arisah-ijeoma** [comment]
got rid of turbolinks

**Comment by emitche** [comment]
Line breaks that users have previously created in their answers are not being respected when Summernote opens the text of the answer. I wonder if there's a way to fix that.

**Comment by arisah-ijeoma** [comment]
I'll branch of this branch and look into that @emitche

**Comment by arisah-ijeoma** [comment]
@emitche, I have a fix out ready for testing in #543 
it should be merged into this branch before the branch is merged into master

**Pull request merged**